newIDE/app/src/ProjectsStorage/CloudStorageProvider/CloudResourceFetcher.js

```diff
@@ -103,6 +103,7 @@ export const moveUrlResourcesToCloudFilesIfPrivate = async ({
   };
 
   const resourcesToFetchAndUpload = getResourcesToFetchAndUpload(project);
+  if (resourcesToFetchAndUpload.length === 0) return result;
```

Comment:
Interesting, what is the purpose exactly of returning early? Is this visible for the user?

Reply:
It was an issue because the next instruction was called even though there is no resource to fetch, but this instruction makes an API call to our backend to an endpoint that doesn't need to be adapted so that the admin can call it.
Overall, it will avoid making some API calls on each cloud project opening.
So nothing visible to the user